requirements.txt

```diff
@@ -3,7 +3,8 @@ ipython>=5.5.0
 ipywidgets>=7.6.5  # required by pycaret.internal.display
 tqdm>=4.62.0  # required by pycaret.internal.display
 numpy>=1.21, <1.27
-pandas>=1.3.0, <2.0.0  # Can't >=2.0.0 because of sktime
+pandas>=1.3.0, <2.1.0
+# pandas>=2.1.0; python_version >= "3.9"
```

Comment:
why this line? we no longer support Python 3.8 so this is obsolete.

Reply:
I think python 3.7 is obsolete, but we support python 3.8.
Pandas 2.1 only supports at minimum python 3.9, then who have python 3.8 cannot use pandas 2.1 only 2.0

Reply:
you are right, we do support 3.8, I was confused with another project. this line can be removed then instead of commented.